### PR TITLE
os/wqueue : Change global variable to static

### PR DESCRIPTION
**Pull request by YiHyunJin**
Change g_usrsem, g_usrmutex, g_hpwork, g_lpwork and g_usrwork to static.
So added get function for using in another file.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [07335e74405c3c4cc76b4072f015a8037b073f3c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [07335e74405c3c4cc76b4072f015a8037b073f3c] - Code Rule Check OK.

File: os/wqueue/work_signal.c [hunk 1]
```diff
@@ -92,25 +92,29 @@ int work_signal(int qid)
...
+		hp_work = get_hpwork();
+		pid = hp_work->worker[0].pid;
 	} else
 #endif
 #ifdef CONFIG_SCHED_LPWORK
 	if (qid == LPWORK) {
 		int wndx;
 		int i;
-
+		struct lp_wqueue_s *lp_work;
```
Comment (sunghan-chang) [inline os/wqueue/work_signal.c]: miss get_lpwork()

**Comment by seinfra** [comment]
Target : [1c0ea8ecb7f52a912caae95f2738771395321360] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1c0ea8ecb7f52a912caae95f2738771395321360] - Code Rule Check OK.

File: os/wqueue/work_signal.c [hunk 2]
```diff
@@ -92,25 +92,29 @@ int work_signal(int qid)
...
 		/* Find an IDLE worker thread */
-
+		lp_work = get_lpwork();
```
Comment (Hyunjun85) [inline os/wqueue/work_signal.c]: How about do you initialize the pointer as NULL. and after assign any pointer value.
Or when you declare the pointer variable, you can assign any poiniter value. 

Anyway, like this implementation can get some warnings from svace or coverity.

**Comment by seinfra** [comment]
Target : [8b96d0e4605e31f6febfa46cc9e3336ec285868b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8b96d0e4605e31f6febfa46cc9e3336ec285868b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [38249ba6c58679d3381f18748e95918780198d50] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [38249ba6c58679d3381f18748e95918780198d50] - Code Rule Check OK.

File: os/wqueue/work_signal.c (hunk 2 above)
Comment (YiHyunJin) [reply os/wqueue/work_signal.c]: Okay,  I initialized it to NULL

**Comment by seinfra** [comment]
Target : [b31b0086abef527215cfe50c9ce22cbaa7e8eed8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b31b0086abef527215cfe50c9ce22cbaa7e8eed8] - Code Rule Check OK.

**Pull request closed without merging**